Signal/src/view controllers/CallInterstitialViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+@objc(OWSCallInterstitialViewController)
+class CallInterstitialViewController: UIViewController {
+
+    let TAG = "[CallInterstitialViewController]"
+
+    var wasCallCancelled = false
+    var callToken: String?
+
+    // MARK: Views
+
+    var hasConstraints = false
+    var blurView: UIVisualEffectView!
+    var contentView: UIView!
+
+    // MARK: Initializers
+
+    required init?(coder aDecoder: NSCoder) {
+        assert(false)
+        super.init(coder: aDecoder)
+    }
+
+    required init() {
```

Comment:
Hah.  Sadly, I doubt I'll ever find the time to remove it, but I absolutely intend to do all new views programmatically.